recipes/tinycbor/meta.yaml

```
@@ -0,0 +1,90 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+# If your package is python based, we recommend using Grayskull to generate it instead:
+# https://github.com/conda-incubator/grayskull
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "tinycbor" %}
+{% set version = "0.6.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # If getting the source from GitHub, remove the line above,
+  # uncomment the line below, and modify as needed. Use releases if available:
+  #url: https://github.com/intel/tinycbor/releases/download/{{ version }}/tinycbor-{{ version }}.tar.gz
+  # and otherwise fall back to archive:
+  url: https://github.com/intel/tinycbor/archive/v{{ version }}.tar.gz
+  sha256: 512e2c9fce74f60ef9ed3af59161e905f9e19f30a52e433fc55f39f4c70d27e4
+  # sha256 is the preferred checksum -- you can get it for a file with:
+  #  `openssl sha256 <file name>`.
+  # You may need the openssl package, available on conda-forge:
+  #  `conda install openssl -c conda-forge``
+  patches:
+    - 0001-Add-CMake-support.patch
+    - 0002-Fix-static_assert-compiler-support.patch
+
+build:
```

```suggestion
build:
  run_exports:
    - {{ pin_subpackage( name|lower, max_pin='x.x.x' ) }}
```
Please add a run export that chooses the appropriate API package version for this packages ABI compatability guarantees.